plugins/kubernetes-backend/src/service/KubernetesFetcher.ts

```diff
@@ -29,13 +29,9 @@ import {
   FetchResponse,
   KubernetesFetchError,
   KubernetesErrorTypes,
+  PodStatusFetchResponse,
 } from '@backstage/plugin-kubernetes-common';
 import { KubernetesClientProvider } from './KubernetesClientProvider';
-import { PodStatus } from '@kubernetes/client-node/dist/top';
-
-export interface Clients {
```

Comment:
I removed this interface as it's not used anywhere, I hope it won't cause any problems 😳 .